package.json

```diff
@@ -80,63 +80,29 @@
         "title": "Tabulous: Reload default terminals"
       },
       {
-        "command": "tabulous.showTerminal1",
```

Comment:
deleting these are a breaking change

Reply:
True, didn't want to commit that. Was experiencing with using a single command with argument instead but didn't turn out to play well with keybindings so I dropped the idea.